src/modules/auth/resources/ResourcesDetailContainer.js

```diff
@@ -53,114 +57,149 @@ const ResourceDetailContainer = (props) =>  {
+			if (response.data.result !== "OK") {
+				throw new Error(t("ResourcesDetailContainer|Failed to delete resource"));
+			}
+			props.app.addAlert("success", t("ResourcesDetailContainer|Resource successfully deleted"));
+			// redirect to list of deleted resources
+			props.history.push("/auth/resources-deleted");
+		} catch(e) {
+			console.error(e);
+			props.app.addAlert("warning", `${t("ResourcesDetailContainer|Failed to delete resource")}. ${e?.response?.data?.message}`, 30);
+		}
+	}
 
-							<CardFooter>
-								<ButtonGroup>
+	return (
+		<Container className="resource-container">
+			<Card className='resource-detail-description'>
+				<Form onSubmit={handleSubmit(onSubmit)}>
+					<CardHeader className="border-bottom">
+						<div className="card-header-title">
+							<i className="cil-lock-unlocked pr-2"></i>
+							{t("ResourcesDetailContainer|Resource")}
+						</div>
+					</CardHeader>
+
+					<CardBody>
+						<Row>
+							<Col md={3}>{t("Name")}</Col>
+							<Col>
+								{ editMode ?
+									<Input
+										id="resource_name"
+										name="resource_name"
+										type="text"
+										autoComplete="off"
+										onChange={registerName.onChange}
+										onBlur={registerName.onBlur}
+										innerRef={registerName.ref}
+									/>
+								:
+									resource?._id
+								}
+							</Col>
+						</Row>
+						<Row className="mt-3">
+							<Col md={3}>{t("Created at")}</Col>
+							<Col><DateTime value={resource?._c} /></Col>
+						</Row>
+						<Row>
+							<Col md={3}>{t("Modified at")}</Col>
+							<Col><DateTime value={resource?._m} /></Col>
+						</Row>
+
+							<FormGroup row className="mt-3">
+								<Col sm={3}>{t("Description")}</Col>
+								<Col sm={6} className="resource-detail-description">
 									{editMode ?
-										<>
-											<Button color="primary" type="submit" >{t("Save")}</Button>
-											<Button color="outline-primary" type="button" onClick={(e) => (setEditMode(false), setOnUpdate(false))}>{t("Cancel")}</Button>
-										</>
-									:
-										<ButtonWithAuthz
-											color="primary"
-											outline
-											type="button"
-											onClick={(e) => (e.preventDefault(), setEditMode(true))}
-											resources={resources}
-											resource={resourceEdit}
-										>
-											{t("Edit")}
-										</ButtonWithAuthz>
-									}
+										<Input
+											id="resource_description"
+											name="resource_description"
+											type="textarea"
+											rows={3}
+											autoComplete="off"
+											onChange={registerDescription.onChange}
+											onBlur={registerDescription.onBlur}
+											innerRef={registerDescription.ref}
+										/>
+									: resource?.description }
+								</Col>
+							</FormGroup>
+					</CardBody>
+
+					<CardFooter>
+						{editMode ?
+							<>
+								<ButtonGroup>
+										<Button color="primary" type="submit" >{t("Save")}</Button>
```

Comment:
@petrKavulok I bet this button should be `ButtonWithAuthz` with some particular resource